### PR TITLE
Feat/update project info

### PR DESCRIPTION
**Pull request by garnetred**
Small text change adding more detail to projects. 

## Background
The Clients Project section doesn't have a lot of detail, so I added more information. I also realized that I should move the client project above my personal project (since I've worked on the latter through my business), so I went ahead and made that change. 

## Testing
Run locally using `npm run develop`, verify that the client project appears above the personal project in the Portfolio section. You can also verify that the text has been updated. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other

## Checklist:

- [x] I have self-reviewed my code and added comments where relevant.
- [ ] I have updated any relevant documentation.

### CODE REVIEW

File: content/sections/projects/projects.json [hunk 1]
```diff
@@ -1,5 +1,17 @@
 {
   "projects": [
+    {
+      "visible": true,
+      "category": "Client Projects",
+      "title": "Software Project",
+      "description": "This legacy application filters through multiple CSV files to find matches across files. In my work with this client, I've created an entire test suite from scratch using Jest, identified bugs and edge cases, improved web performance and scaled the application to work with larger datasets, and improved privacy by creating mock data.",
```
Comment (garnetred) [inline content/sections/projects/projects.json:7]: I moved this above the personal project and added a more detailed description.
